mod/developers/classes/Elgg/DevelopersPlugin/Hooks.php

```diff
@@ -44,6 +44,9 @@ public static function alterMenuSections($hook, $type, $output, $params) {
 			return;
 		}
 
+		$output = str_replace('elgg-menu-item-has-toggle', '', $output);
+		$output = str_replace('elgg-menu-closed', '', $output);
```

Comment:
Is there a menu hook I could've used to make these non-toggle menus (instead of hacking the HTML output)?

Also, before I made this change, is there a reason the toggles didn't work in the colorbox? Do modules need to be required by the menu items?